packages/loot-core/src/platform/server/sqlite/index.web.js

```diff
@@ -162,6 +162,9 @@ export async function openDatabase(pathOrBuffer) {
       PRAGMA journal_mode=MEMORY;
       PRAGMA cache_size=-10000;
     `);
+      db.create_function("regexp", function (regex, text) {
+        return new RegExp(regex).test(text) ? 1 : 0;
```

Comment:
Small suggestion: improve unicode support (so emoji count as `.` rather than `..`).

```suggestion
        return new RegExp(regex, "u").test(text) ? 1 : 0;
```

Reply:
Good idea. I also added the ignore case flag as that seems to align with how contains is set up.

![image](https://user-images.githubusercontent.com/2128234/224454495-97296876-9dd7-41dc-90c5-70e73c324ef3.png)